include/bluetooth/conn.h

```diff
@@ -129,13 +129,21 @@ struct bt_conn_le_info {
 	u16_t interval; /** Connection interval */
 	u16_t latency; /** Connection slave latency */
 	u16_t timeout; /** Connection supervision timeout */
+	u8_t *features; /** Remote device's ble features */
```

Comment:
Declare this one as const u8_t * features, this will allow it to be assigned to a const object. Which is the case in:
int bt_conn_get_info(**const** struct bt_conn *conn, ...)

Reply:
Also we should give some information about the length. I think we should create a `#define BT_LE_FEATURE_LEN 8` and reference that here.